README.md

```diff
@@ -2,19 +2,19 @@
   <img alt="githubdark-logo" src="https://rawgit.com/StylishThemes/logos/master/github.dark/githubdark-mini.svg" width="580">
   <br>
   <a href="https://github.com/StylishThemes/GitHub-Dark/tags">
-    <img src="https://img.shields.io/github/tag/StylishThemes/GitHub-Dark.svg?label=tag" alt="Tag">
+    <img src="https://img.shields.io/github/tag/StylishThemes/GitHub-Dark.svg?label=version&style=flat" alt="Version">
   </a>
   <a href="https://github.com/StylishThemes/GitHub-Dark/stargazers">
-    <img src="http://github-svg-buttons.herokuapp.com/star.svg?user=StylishThemes&repo=GitHub-Dark&style=flat&background=007ec6" alt="Star">
+    <img src="http://github-svg-buttons.herokuapp.com/star.svg?user=StylishThemes&repo=GitHub-Dark&style=flat&background=007ec6" alt="Stars">
```

Comment:
I don't suppose we have a "stars" badge available somewhere? This singular always bugged me.

Reply:
Sorry not that Im aware.

The singular along with font fixes is up to @ddavison, on the stars badge, a suggestion, he could add a ?label=stars override or whatever similar to what shieldsIO does

I could use the shieldsIO stars badge (and use plural) but I dont like the rounded way they count 2896 = 3K, may as well invent a more arbitrary number like 9999999 stars, hahaha